casthour.html

```diff
@@ -1660,10 +1660,6 @@
 						        },
 						    ]
 						},
-						{
```

Comment:
Ah shit, good catch ;_;

Reply:
Lol welp thats now in the commit history forever.

Reply:
Well look, if you're able to noodle around in the commit history for these things, you're able to just use a fork that has them, so it's not the worst thing in the world.